R/make-array.R

```diff
@@ -1,10 +1,12 @@
 #' Make a Categorical Array or Multiple Response variable
 #'
-#' Array variables are composed of a set of "subvariables" bound
-#' together for display in the app. For example, you might have a set of
-#' survey questions that ask how the respondent would rate a tv show from
-#' 1-5. Array variables allow you to display all of their ratings in a compact
-#' table rather than a set of distinct variables.
+#' We strongly recommend using `deriveArray` which leaves your subvariables in
```

Comment:
This is a little overly harsh, there are valid reasons to use `makeArray()` (just not in how YouGov uses crunch). Can you change to, "In most situations we recommend using `deriveArray()`"?